btree/btree.c

```diff
@@ -3358,12 +3358,9 @@ static int64_t btree_get_tick(struct m0_sm_op *smop)
 				node_put(oi->i_level[i].l_node);
 				oi->i_level[i].l_node = NULL;
 			}
-			if (oi->i_level[i].l_alloc != NULL) {
-				node_put(oi->i_level[i].l_alloc);
-				oi->i_level[i].l_alloc = NULL;
-			}
 		}
-		free(bop->bo_i->i_level);
+		m0_free(bop->bo_i->i_level);
```

Comment:
We have alloc function allocating memory bop->bo_i and also bop->bo_i->i_level. Similarly a free function which releases the memory would make sense.
P.S.: Radha has changed the name of alloc to alloc_levels. Please sync with her to get the appropriate name for free.

Reply:
Added a level_cleanup() function to clean up levels and free memory.